script/test

```diff
@@ -4,4 +4,4 @@ set -e
 cd "$(dirname "$0")/.."
 source script/_common
 
-docker_bundle_exec rspec "$@"
+docker_bundle_exec bundler rspec "$@"
```

Comment:
I don't understand why `bundler` is needed here?

Since `docker_bundle_exec` already calls `docker bundle exec`, this ends up as `docker bundle exec bundler rspec...` so isn't the extra `bundler` one too many?

Reply:
Ha, that's a bit confusing isn't it? The first parameter to docker_bundle_exec is now the ecosystem image to build/use. So it's saying build the bundler ecosystem, then run rspec in it.